x-pack/plugins/alerting/server/alerts_client/legacy_alerts_client.test.ts

```diff
@@ -4,74 +4,68 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import sinon from 'sinon';
```

Comment:
Updated these tests so that they're not just testing that the right helper functions get called but also inputs and outputs.